packages/mu-local/driver.js

```diff
@@ -61,3 +73,6 @@ module.exports = function createLocalDriver (options) {
     }
   }
 }
+
+module.exports.forceError = function forceError () { ERROR_ON_SEND = true }
```

Comment:
can we stick with vertical state - ie have an `opts.forceError` - also why are we doing this? purely for tests? if purely for tests is there another way?